src/main/java/Duke.java

```diff
@@ -1,10 +1,275 @@
+            isValidCommand = false;
+        }
+
+        if (isValidCommand) {
+            printLine();
+            System.out.println("Got it bro. I've added this task:");
+            System.out.println(tasks.get(tasks.size() - 1));
+            System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+            printLine();
+        }
+    }
+
+    private static void handleUnknownCommand(String input) throws DukeInvalidArgumentException {
+        printLine();
+        System.out.println("Huh? " + input + "?");
+        System.out.println("You haven't read the documentation, have you?");
+        printLine();
+        throw new DukeInvalidArgumentException();
+    }
+
+    private static void addTodo(String input) throws DukeInvalidArgumentException {
+        String description;
+        try {
+            description = input.trim().split("todo")[1].trim();
+        } catch (IndexOutOfBoundsException e) {
+            printLine();
+            System.out.println("What task do you have? Add the task description please...");
+            printLine();
+            throw new DukeInvalidArgumentException();
+        }
+        tasks.add(new Todo(description));
+    }
+
+    private static void addEvent(String input) throws DukeInvalidArgumentException {
+        String eventDetails;
+        try {
+            eventDetails = input.trim().split("event")[1];
+        } catch (IndexOutOfBoundsException e) {
+            printLine();
+            System.out.println("Come on bro you must specify the event details...");
+            printLine();
+            throw new DukeInvalidArgumentException();
+        }
+        String description, at;
+        try {
+            description = eventDetails.split("/at")[0].trim();
+            at = eventDetails.split("/at")[1].trim();
+        } catch (IndexOutOfBoundsException e) {
+            printLine();
+            System.out.println("Forgot to type something?");
+            System.out.println("I need both the description and the time to add an event.");
+            printLine();
+            throw new DukeInvalidArgumentException();
+        }
+        tasks.add(new Event(description, at));
+    }
+
+    private static void addDeadline(String input) throws DukeInvalidArgumentException {
+        String deadlineDetails;
+        try {
+            deadlineDetails = input.trim().split("deadline")[1];
+        } catch (IndexOutOfBoundsException e) {
+            printLine();
+            System.out.println("Come on bro you must specify the deadline details...");
+            printLine();
+            throw new DukeInvalidArgumentException();
+        }
+        String description, by;
+        try {
+            description = deadlineDetails.split("/by")[0].trim();
+            by = deadlineDetails.split("/by")[1].trim();
+        } catch (IndexOutOfBoundsException e) {
+            printLine();
+            System.out.println("Forgot to type something?");
```

Comment:
Maybe you can abstract these print statements (line 120, 121, 96, 97) since only the last 2 words are different (...add a deadline vs ...add an event)

I think they'll look neater, with your print statements declared all at the top as string constants - so whenever you wanna change the print outputs, you dont have to scroll through your code to locate them)